src/tree/CosmosDBAccountWizard/CosmosDBAccountCreateStep.ts

```diff
@@ -31,19 +31,19 @@ export class CosmosDBAccountCreateStep extends AzureWizardExecuteStep<ICosmosDBW
             locations: [{ locationName: locationName }],
             kind: defaultExperience.kind,
             // Note: Setting this tag has no functional effect in the portal, but we'll keep doing it to imitate portal behavior
-            tags: { defaultExperience: defaultExperience.tag },
+            tags: { defaultExperience: nonNullProp(defaultExperience, 'tag') },
             capabilities: <Capability[]>[]
         };
 
         if (defaultExperience.capability) {
             options.capabilities.push(<Capability>{ name: defaultExperience.capability });
         }
 
-        wizardContext.databaseAccount = await client.databaseAccounts.createOrUpdate(rgName, accountName, options);
+        wizardContext.databaseAccount = await client.databaseAccounts.createOrUpdate(rgName, serverName, options);
 
         // createOrUpdate always returns an empty object - so we have to get the DatabaseAccount separately
-        wizardContext.databaseAccount = await client.databaseAccounts.get(rgName, accountName);
-        ext.outputChannel.appendLog(`Successfully created Cosmos DB account "${accountName}".`);
+        wizardContext.databaseAccount = await client.databaseAccounts.get(rgName, serverName);
+        ext.outputChannel.appendLog(`Successfully created Cosmos DB server "${serverName}".`);
```

Comment:
Right so the Cosmos DB-specific messages (like in this file) should probably still use "account". I don't care about the internal code (variable names, etc.) - just the user-visible strings

Reply:
What about this message in SubscriptionTreeItem.ts `vscode.window.showInformationMessage(`Successfully created server "${resourceName}".`);`
This applies to both Cosmos and Postgres. Should we have separate messages for both or keep it as it is with "server" in the string?

Reply:
The correct thing would be to have separate messages when possible. This file is easy to change so I commented on it. That example is a little more annoying to change, so I don't care as much